src/renderer/containers/Account/CreateAccountModal/index.tsx

```diff
@@ -91,7 +91,10 @@ const CreateAccountModal: FC<ComponentProps> = ({close, isGetStartedModal = fals
 
   const validationSchema = useMemo(() => {
     return yup.object().shape({
-      nickname: yup.string().notOneOf(managedAccountNicknames, 'That nickname is already taken'),
+      nickname: yup
+        .string()
+        .notOneOf(managedAccountNicknames, 'That nickname is already taken')
+        .max(64, 'Nickname must not be more than 64 characters.'),
```

Comment:
Isn't it better to define const for this variable and use it everywhere? Also, these error/warning strings can be made const as well. These can be defined under form-validation.ts as we have it for signing key validation (SIGNING_KEY_LENGTH_ERROR)

Reply:
Do you want me to define const for `That nickname is already taken` and other warning strings within this PR as well?

Reply:
It would have been better to keep constants related to the current issue in this PR itself and a new issue to define the constants for existing warning/error strings; and since these are not critical changes and you have already reverted it, this is fine as well.